.github/workflows/sphinx.yml

```diff
@@ -34,21 +36,23 @@ jobs:
       contents: write
 
     steps:
-      - name: Checkout Repo
+      - name: Checkout Repo ⚡️
         uses: actions/checkout@v3
 
-      - name: Set Up Python
+      - name: Set Up Python 🐍
         uses: actions/setup-python@v3
         with:
-          python-version: "3.9"
+          python-version: 3.9
 
-      - name: Install Dev Dependencies
+      - name: Install Doc Dependencies 📦
         run: |
-          python -m pip install --upgrade pip
-          make install-docs-env
+          python -m pip install doit plumbum
```

Comment:
I think it would be easier to put these both in `requirements-docs.txt` as well and instead of running `python -m doit setup_docs`, directly run `python -m install -r requirements-docs.txt`.

Reply:
agree - was experimenting with this but it makes sense to keep the whole workflow as simple as possible